e2e/atlas/clusters_test.go

```diff
@@ -30,11 +29,7 @@ import (
 )
 
 func TestAtlasClusters(t *testing.T) {
-	cliPath, err := filepath.Abs("../../bin/mongocli")
-	if err != nil {
-		t.Fatalf("unexpected error: %v", err)
-	}
-	_, err = os.Stat(cliPath)
+	_, err := os.Stat(cliPath)
```

Comment:
[nit] this is a test to make sure the binary exists in the path, you could move this to the same `init` function where you do many of the cli path discovery